combat-trainer.lic

```diff
@@ -652,6 +652,7 @@ class LootProcess
     echo "Attempting necromancer ritual #{ritual} on #{mob_noun}" if $debug_mode_ct
 
     if ritual.eql?('butcher')
+      do_necro_ritual(mob_noun, 'preserve', game_state)
```

Comment:
💡 What do you think about instead moving the `butcher` check after line 663 and adding `butcher` to the array on line 663?

Reply:
This is more a question for @izawwlgood or other Necros. I'm so far detached from what is good/proper. :(

Reply:
Seems like this would be better as optional rather than required, could add a lot of RT and maybe isn't required, these are my first thoughts, just no real foundation to make any accurate statements beyond that.

Reply:
Just double checked this, doesn't appear to be needed! 

The butcher_corpse method handles this already!

```
  def butcher_corpse(mob_noun, ritual, game_state)
    return unless ritual.eql?('butcher')
    echo "Butchering the #{mob_noun}'s corpse!" if $debug_mode_ct
    echo ' Only butchering it once!' if $debug_mode_ct && @dissect_and_butcher && !@ritual_type.eql?('butcher')

    do_necro_ritual(mob_noun, 'preserve', game_state)
    @equipment_manager.stow_weapon(game_state.weapon_name)
```